services/machine_learning/project/csv_parser.py

```diff
@@ -29,16 +28,19 @@ def extract_columns(app, auth_token, file_names) :
         insert_cwd(app, auth_token, abs_path)
 
     #download files from s3. Make a get request from s3 endpoint
-    g_url = "http://file_system:5000/s3/download"
+    g_url = "http://file_system:5000/s3/downloadUploaded"
     g_headers = {"Authorization" : 'Bearer ' + auth_token}
     #get only the data for first file in file name list
     if not file_names:
         print("List of given file names is empty")
     else:
         f = file_names[0]
-        g_param = {"address" : f}
-        r = requests.get(url = g_url, headers = g_headers, params = g_param)
-        #not sure if it returns download url or zip file of all files? Need to ask Jake
+        g_param = {f : f}
```

Comment:
While fine for right now, note that this will need to be changed if multiple files are used